server/lib/sync-agent.js

```diff
@@ -103,7 +110,18 @@ export default class SyncAgent {
       return false;
     }, {});
 
-    return !!conn;
+    const hasRequiredFields = !!conn;
+
+    if (hasRequiredFields) {
+      if (this.adapter.in.isValidConfiguration) {
+        if (!this.adapter.in.isValidConfiguration(settings)) {
```

Comment:
What happens exactly when the operator provides wrong or partial access settings?
That would be good story for the status feedback.

Reply:
This was the weirdest one because if you provide an account name which also includes the region like this: js12355-us-east-1, it completely kills the connector where it won't run anymore.

Reply:
so we actually have to validate the "value" of the connector settings